src/mpb.cpp

```diff
@@ -173,7 +185,7 @@ complex<double> eigenmode_amplitude(void *vedata, const vec &p,
 #undef D
 
   return (complex<double>(double(real(ret)), double(imag(ret)))
-	  * amp_func(p));
+	  * amp_func(p)) * std::polar(1.0, TWOPI*phase);
```

Comment:
It turns out that by simply negating the phase (i.e., `-TWOPI*phase`) the results from `get_eigenmode` (based on the latest commit `whoops`) are once again sinusoidal. A side-by-side comparison of the results from the `amp_func` source and `get_eigenmode` are shown below for 0°, 10.7°, 22.9°. The wavelength of the planewaves from `get_eigenmode` for non-zero `k` are *exactly* double those from the `amp_func` source.

Perhaps an extra factor of `2` was included somewhere when it should not have been?

![ez_real_ampfunc_geteigenmode_vary_theta](https://user-images.githubusercontent.com/7152530/46050242-c501ad80-c0e7-11e8-9cad-7fc7685288de.png)